x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices.tsx

```diff
@@ -80,21 +85,26 @@ export const EngineIndices: React.FC = () => {
     },
     type: 'icon',
   };
+
   const columns: Array<EuiBasicTableColumn<EnterpriseSearchEngineIndex>> = [
     {
-      field: 'name',
       name: i18n.translate('xpack.enterpriseSearch.content.engine.indices.name.columnTitle', {
         defaultMessage: 'Index name',
       }),
-      render: (name: string) => (
-        <EuiLinkTo
-          data-test-subj="engine-index-link"
-          to={generateEncodedPath(SEARCH_INDEX_PATH, { indexName: name })}
-        >
-          {name}
-        </EuiLinkTo>
-      ),
-      sortable: true,
+      render: ({ health, name }: EnterpriseSearchEngineIndex) =>
+        health === 'unknown' ? (
+          <EuiLink onClick={() => {}} color="subdued">
```

Comment:
why return an empty `EuiLink` here vs just rendering the `{name}`?

Reply:
hmm i was trying to match the styling that i eyeballed in [these designs](https://github.com/elastic/enterprise-search-team/issues/3767#issuecomment-1419272698) but it is a bit odd, huh?

Reply:
Yea, I think it's bad for accessibility too. I wouldn't render a link unless we want it to be a link.